backend/src/main/java/kr/touroot/travelogue/service/TravelogueCountryService.java

```diff
@@ -19,30 +19,33 @@ public class TravelogueCountryService {
 
     private final TravelogueCountryRepository travelogueCountryRepository;
 
-    @Transactional(readOnly = true)
-    public List<TravelogueCountry> readCountryByTravelogue(Travelogue travelogue) {
-        return travelogueCountryRepository.findAllByTravelogue(travelogue);
-    }
-
     @Transactional
-    public void createTravelogueCountries(Travelogue travelogue, TravelogueRequest request) {
+    public List<TravelogueCountry> createTravelogueCountries(Travelogue travelogue, TravelogueRequest request) {
         Map<CountryCode, Long> countryCounts = countCountries(request);
 
-        countryCounts.forEach((countryCode, count) -> travelogueCountryRepository.save(
-                new TravelogueCountry(travelogue, countryCode, count.intValue())));
+        return countryCounts.entrySet().stream()
+                .map(entry -> travelogueCountryRepository.save(
+                        new TravelogueCountry(travelogue, entry.getKey(), entry.getValue().intValue())))
```

Comment:
```suggestion
                .map(entry -> travelogueCountryRepository.save(
                        new TravelogueCountry(travelogue, entry.getKey(), entry.getValue().intValue()))
                 )
```
괄호가 조금 이상하게 되어있는거 같아요!